python/ynab_client.py

```diff
@@ -65,6 +67,3 @@ def findPreviousTransaction(payee_name):
         .filter(Transaction.imported_payee==payee_name)\
         .order_by(Transaction.date.desc())\
         .first()
-
-# Sync with YNAB
-sync()
```

Comment:
Potentially yeah. I am not sure if the client will be imported at that point though. I would need to experiment with how python loads things.